INSTALL.md

```diff
@@ -14,20 +14,50 @@ Starting with the next ponyc release, you'll also be able to download prebuilt p
 
 Prebuilt Linux packages are available via [ponyup](https://github.com/ponylang/ponyup) for Glibc and musl libc based Linux distribution. You can install nightly builds as well as official releases using ponyup.
```

Comment:
We should specify that we only have prebuilt images for AMD64 at this time

Reply:
Can you make a suggest a change for that and I'll incorporate?

Reply:
Actually that feels like a different PR, agreed? As it applies in general across all platforms.

Reply:
I guess you're correct, since macOS and Windows will likely have more widespread ARM support soon.